src/phpDebug.ts

```diff
@@ -1008,7 +969,6 @@ class PhpDebugSession extends vscode.DebugSession {
             this.sendErrorResponse(response, error)
```

Comment:
This line is not valid anymore because the response has already been sent.

Reply:
@felixfbecker I ended up reverting this part to how it was.  Really the docs are rather vague so we are forced to figure out whether we should send immediately or wait.  My original thought was to not wait since the response has no information to tell vscode, it is just an acknowledgment.

But now I'm thinking, this is a disconnect response, there is no harm in waiting, especially since it already has timeouts built in.

In addition, this could be a part of a restart request, in which case we do definitely want to wait before telling vscode, I could see a situation where we let vscode know too early and it results in vscode attempting to start a new connection before the old one is done.  You would end up with errors about the port already being in use.

**tl;dr:** No harm in making it work like it did before, and there is potentially a problem by not waiting, so better to be on the safe side since we don't have much direction from the docs.